sims/xcelium/Makefile

```diff
@@ -0,0 +1,138 @@
+#########################################################################################
+# xcelium makefile
+#########################################################################################
+
+define CAD_INFO_HEADER
+# --------------------------------------------------------------------------------
+# This script was written and developed by Chipyard at UC Berkeley; however, the
+# underlying commands and reports are copyrighted by Cadence. We thank Cadence for
+# granting permission to share our research to help promote and foster the next
+# generation of innovators.
+# --------------------------------------------------------------------------------
+endef
+
+export CAD_INFO_HEADER
+
+#########################################################################################
+# general path variables
+#########################################################################################
+base_dir=$(abspath ../..)
+sim_dir=$(abspath .)
+
+#########################################################################################
+# include shared variables
+#########################################################################################
+include $(base_dir)/variables.mk
+
+#########################################################################################
+# name of simulator (used to generate *.f arguments file)
+#########################################################################################
+sim_name = xrun
+
+#########################################################################################
+# xcelium simulator types and rules
+#########################################################################################
+sim_prefix = simx
+sim = $(sim_dir)/$(sim_prefix)-$(MODEL_PACKAGE)-$(CONFIG)
+sim_debug = $(sim)-debug
+sim_workdir = $(sim).xcelium.d
```

Comment:
Should this be located in the generated sources like Verilator's build directory?

Reply:
Similar for the TCL scripts.

Reply:
Fixed